apps/Smart/Cleanarr/code.js

```diff
@@ -0,0 +1,31 @@
+﻿class Cleanarr {
+    doFetch(args, endpoint) {
+        return args.fetch({
+            url: endpoint,
+            timeout: 18000
+        });
+    }
+
+    async status(args) {
+        let data = await this.doFetch(args, 'content/dupes');
+
+        let count = data?.length;
```

Comment:
add some null checking here, data could be null (which you're checking for here) but then no later on in the for loop.  and filedata.media could be null, then parts etc.  just avoid any crashes